changelogs/fragments/79187--wheel-0.38.0.yml

```
@@ -0,0 +1,4 @@
+bugfixes:
+  - >-
+    Restrict ``wheel`` below v0.38.0 in ``pip`` integration test when
+    running under Python 3.6 and older.
```

The changelog should reflect that this is a change in ansible-test rather than a specific integration test:

```suggestion
    - ansible-test - Add ``wheel < 0.38.0`` constraint for Python 3.6 and earlier.
```